examples/vue-todos/src-tauri/tauri.linux.conf.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we _need_ these 3 files? Might as well just use the general conf file 🤷
i'd be fine with keeping these as it shows that per-platform configs are possible. But this could also cause confusion instead (aka users could think it's _needed_ for the systray). So yeah i'm kinda torn here.

Reply:
Well I haven't seen any activity on my PR for documentation but the solution you're discussing is not documented until this goes through. Further, the icon sizes that the various platforms expect are different, so while I documented what I could see from the outside about the current undocumented work around I don't have any visibility to whether it really acomodates all use cases. To me the safer bet is to have the separate files as this is a good example of something that is different by platform. 

I admit, I had intended to test both but felt a bit squeezed for time.

Reply:
@FabianLars is there a macro I can use to conditionally include blocks of code by OS?

Reply:
> Any more features and we need to move this into its own repo :D

LOL. Still i think really nice "examples" do help uptake of new platforms. That said, I promise no new features.
 
> Anyway, i'll do the actual review separately and answer the things you mentioned here:
> 
> Native menu items not working on linux is fixed on the next branch.

Would it make any sense to move this plugin to the next branch or are we maintaining beta-8 compatibility for now?

> 
> `MenuItem::Quit` not working on Windows is expected, basically none of the native ones work there, see "platform-specific notes" [here](https://docs.rs/tauri/1.0.0-beta.8/tauri/enum.MenuItem.html#variant.Quit)

@FabianLars is there a macro I can use to conditionally include blocks of code by OS? 

> `Show` on windows does work on the next branch too. Can't remember if this was explicitly fixed or just a coincidence.

So "show" is only a feature on the system-tray and both "hide" and "show" seem to work on all platforms on beta-8. It's only the dropdown menu on the main window that acts very oddly on Windows.

Reply:
> LOL. Still i think really nice "examples" do help uptake of new platforms. That said, I promise no new features.

Yeah nothing wrong with in depth examples, well except the debt of keeping them up to date. So, this wasn't meant negatively at all 👍 

> Would it make any sense to move this plugin to the next branch or are we maintaining beta-8 compatibility for now?

Hmm, good question. With the audit changes being so close i'd probably stick with beta.8 for now and switch over to the next versions or the `dev` branch once the changes are there and we don't release RCs.

> So "show" is only a feature on the system-tray and both "hide" and "show" seem to work on all platforms on beta-8. It's only the dropdown menu on the main window that acts very oddly on Windows.

It's about native items vs custom items, like windows just doesn't provide this interface _natively_ and we don't polyfill this for native items either (if that's even possible).
You can have the menu's hide/show work on Windows too if you use a custom item for that (with a similar menu event handler like you use for the systray),